helmfile.d/helmfile-99.post.yaml

```diff
@@ -30,6 +30,12 @@ releases:
       pkg: harbor
       group: jobs
     <<: *jobs
+  - name: job-harbor-backup
+    installed: {{ $a | get "harbor.backup.enabled" }}
```

Comment:
I believe it should be:
```
installed: {{ and ($a.harbor.enabled) ($a | get "harbor.backup.enabled"  false) }}
```

You could also add the `harbor.backup.enabled` to the `defaults.yaml` file then you do not need to use `get`